src/main/java/it/pagopa/pm/gateway/service/VposService.java

```diff
@@ -210,6 +211,15 @@ private PaymentRequestEntity createEntity(String clientId, String mdcFields, Str
         return entity;
     }
 
+    private VposDatabaseRequest generateVposDatabaseRequest(StepZeroRequest request) {
+        VposDatabaseRequest vposDatabaseRequest = new VposDatabaseRequest();
+        vposDatabaseRequest.setIsFirstPayment(request.getIsFirstPayment());
+        vposDatabaseRequest.setAmount(request.getAmount());
+        vposDatabaseRequest.setIdTransaction(request.getIdTransaction());
+        vposDatabaseRequest.setIdPsp(request.getIdPsp());
+        return vposDatabaseRequest;
+    }
```

Comment:
I would add a `@AllArgsConstructor` to `VposDatabaseRequest` and call the constructor instead of using setters, so that if a new field is added it would immediately trigger a compilation error

Reply:
Done https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/211/commits/598bc521fa6823b16895c36262a9cfd9ee8a416b